resources/views/components/newsletter.blade.php

```diff
@@ -1,6 +1,5 @@
-{{-- To do: this causes a blank page on the register layout --}}
+@props(['id' => uniqId('newsletter-'), 'vModel'])
```

Comment:
Can we think of something cleaner for this?

Reply:
I agree can't v-model be passed with {{ $attributes }} ?